core/chain-configs/src/genesis_config.rs

```diff
@@ -337,6 +457,33 @@ impl Genesis {
         hasher.process_genesis(self);
         hasher.finalize()
     }
+
+    pub fn stream_records_with_callback(&self, callback: impl FnMut(StateRecord)) {
```

Comment:
```suggestion
    fn stream_records_with_callback(&self, callback: impl FnMut(StateRecord)) {
```

Seems like it doesn't need to be `pub`?